packages/babel-generator/test/index.js

```diff
@@ -348,6 +348,20 @@ describe("generation", function () {
     const output = generate(type).code;
     expect(output).toBe("(infer T)[]");
   });
+
+  it("comments with same start index", () => {
+    const code1 = "/* leading a */ a();";
```

Comment:
The example of generating multiple statements for the program body in the PR author comment seems like a more likely scenario than what we're testing here. Can we change this to match that?